src/Coordination/KeeperSnapshotManager.h

```diff
@@ -126,6 +130,13 @@ class KeeperSnapshotManager
         return 0;
     }
 
+    std::string getLatestSnapshotPath() const
+    {
+        if (!existing_snapshots.empty())
+            return existing_snapshots.at(getLatestSnapshotIndex());
```

Comment:
Maybe check that we have it -- will get more clear error in case of bugs.

Reply:
Yes, I will fix.